reference/supported-languages.md

```diff
@@ -0,0 +1,30 @@
+# Languages/Formats OpenRewrite supports
+
+OpenRewrite is continuously adding support for new languages and formats. As of June 2023, OpenRewrite supports:
+
+## Programming languages
+
+* Java
+* Groovy/Gradle 
+* Kotlin
+* COBOL (Closed source, commercial feature)
+
+## Data formats
+
+* XML
+* Properties
+* YAML
+* JSON
+* Protobuf
+* Maven POMs (with dependency management)
+
+## Frameworks
+
+Framework developers have created recipes to help migrate and support their frameworks. These include:
+
+* Spring
+* Quarkus
+* Micronaut
+* Kubernetes
```

Comment:
I'd include Jakarta here as well, as that's a common migration users might be looking for these days.
```suggestion
* Jakarta
* Kubernetes
```
I'm somewhat hesitant about including Kubernetes; we haven't explicitly documented how to run such migration recipes, nor do we hear much from users that are using these recipes.